### DISCUSSION TITLE
Support for clojure-mail or other tools to monitor imap mailboxes?

### DISCUSSION THREAD

**Discussion by bzg** [babashka/babashka#1735] [Q&A]
I use https://github.com/owainlewis/clojure-mail for monitoring imap mailboxes (see [Woof](https://github.com/bzg/woof), used on https://tracker.orgmode.org). 

I would love to rewrite Woof using babashka: one script for monitoring/processing a mailbox, another one for exposing data.

I'm frustrated because I don't find a way to use clojure-mail with babashka or an alternative library offering the same features. 

I would love to (1) find such a library and use it with `deps/add-deps` _or_ (2) load a pod. 

I would also love to be reassured that the library is well maintained, because it seems `clojure-mail` does not get that much attention right now.

Thanks for any hint!  And thanks for babashka, it's great.

**Comment by borkdude** [comment]
Hi. There's https://github.com/tzzh/pod-tzzh-mail for sending mail with babashka.

**Reply by bzg** [reply]
Thanks for your answer.

[clojure-mail](https://github.com/owainlewis/clojure-mail) is not about _sending_ emails, it's about inspecting the contents of imap mailboxes and triggering events for incoming emails.

[clojure-mail](https://github.com/owainlewis/clojure-mail) does not seem compatible with GraalVM - I don't know what would be needed to make it compatible with GraalVM and babashka.
